scripts/release/debian/test_deb_package.py

```diff
@@ -12,7 +12,7 @@
 mod_list = [mod for mod in sorted(os.listdir(root_dir)) if os.path.isdir(os.path.join(root_dir, mod)) and mod != '__pycache__']
 
 pytest_base_cmd = '/opt/az/bin/python3 -m pytest -v --forked -p no:warnings --log-level=WARN'
-pytest_parallel_cmd = '{} -n auto'.format(pytest_base_cmd)
```

Comment:
> With `-n auto`, pytest-xdist will use as many processes as your computer has physical CPU cores.
Use `-n logical` to use the number of logical CPU cores rather than physical ones.
- https://pytest-xdist.readthedocs.io/en/stable/distribution.html